CHANGES.rst

```diff
@@ -4,6 +4,7 @@
 
 4.5.0 (unreleased)
 ==================
+- Fix IE issue in /@@user-information?user_id=TestUser for orderedSelectionList
```

Comment:
There ought to be a blank line between the line of `====` and the first list item.

It might also be useful to mention the GitHub issue number.